wpilibNewCommands/src/main/java/edu/wpi/first/wpilibj2/command/Commands.java

```diff
@@ -115,6 +118,17 @@ public static Command waitSeconds(double seconds) {
     return new WaitCommand(seconds);
   }
 
+  /**
+   * Constructs a command that does nothing, finishing after a specified duration.
+   *
+   * @param time after how long the command finishes
+   * @return the command
+   * @see WaitCommand
+   */
+  public static Command waitTime(Measure<Time> time) {
```

Comment:
:thinking: Is `waitTime` a tautology? Should this just be called `wait`, like the unit-safe C++ version?